src/main/java/frc/team2412/robot/subsystem/IntakeSubsystem.java

```diff
@@ -72,7 +73,7 @@ public IntakeSubsystem(WPI_TalonFX motor, DoubleSolenoid intakeSolenoid) {
         intakeSolenoidState = EXTEND;
         intakeMotorState = STOPPED;
 
-        intakeRetract();
+        // intakeRetract();
```

Comment:
Can leave uncommented, as `intakeRetract()` safely handles missing hardware (or should).